### PR TITLE
Client log in page

### PR DESCRIPTION
**Pull request by sammysamx20**
It'd be nice if someone could see my post code and look over it and see why it won't connect.

### CODE REVIEW

**Comment by sammysamx20** [comment]
Also the reason so many files were changed is because i removed the build

**Comment by sammysamx20** [comment]
Ready to merge, turns out the issues were the variable naming issues in the server code.

File: client/src/index.js [hunk 1]
```diff
@@ -3,41 +3,44 @@ import ReactDOM from 'react-dom';
...
-        <Route path="/login">
-          <h1> pls login </h1>
-        </Route>
-        <Route exact path="/">
+      <Router>
         <div>
-          <Link to="/watch">WatchVideo</Link>
+
+
+        <div class="tab">
```
Comment (developerJuiceLozzoc) [inline client/src/index.js:20]: className or else it wont work

File: client/src/index.js [hunk 2]
```diff
@@ -3,41 +3,44 @@ import ReactDOM from 'react-dom';
...
+          <Link className="tablinks" to="/">Home</Link>
+          <Link className="tablinks"  to="/watch">Watch</Link>
+          <Link className="tablinks"  to="/login">Login</Link>
```
Comment (developerJuiceLozzoc) [inline client/src/index.js:23]: This seems that it goes against our design. the homepage is the log in page, not a tab to be directed, and what happens when they log on? the tab would have to change to log out.

File: client/src/components/LoginPage.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+            <strong>Please Create a Username or Log In </strong>
+          </Panel.Header>
+          <TextField
+            style={style}
+            placeholder='Username...'
+            value={this.state.username}
+            onChange={username => this.setState({ username })}
+          />
+          <Panel.Footer>
+            <Button>Log in</Button>
```
Comment (developerJuiceLozzoc) [inline client/src/components/LoginPage.js:103]: oh no wonder, you dont have anything hooked up to the login button

File: client/src/components/LoginPage.js [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+  getUsernames(){
+    fetch('/api/names/users', {
+      method: 'GET',
+      headers: {
+      'Content-Type': 'application/json',
+      },
+    }).then((response) => {
+      if (!response.ok) {
+          this.setErrorMessage(response.json().content);
```
Comment (developerJuiceLozzoc) [inline client/src/components/LoginPage.js:47]: This is weird, whenever you go into a async funciton, the this word changes. so i always create a `const self= this` as a global varible to set state before going into any fetches. but this seems to work on the client.

File: client/src/components/LoginPage.js (hunk 3 above)
Comment (developerJuiceLozzoc) [reply client/src/components/LoginPage.js:103]: i was wondring why the button didnt work lol

File: client/src/components/LoginPage.js (hunk 4 above)
Comment (JaredBeale) [reply client/src/components/LoginPage.js:47]: I think using `=>` binds `this` and that's why it works.

File: client/src/components/LoginPage.js (hunk 3 above)
Comment (JaredBeale) [reply client/src/components/LoginPage.js:103]: Sam will be adding functionality for this in later PR

File: client/src/index.js (hunk 2 above)
Comment (JaredBeale) [reply client/src/index.js:23]: Sam will be re-organizing the page layout in later PR

File: client/src/index.js (hunk 1 above)
Comment (JaredBeale) [reply client/src/index.js:20]: Sam will fix this in later PR

**Pull request merged**